ontology/yaml/resources/TRANSPORT/entity_types/ABSTRACT.yaml

```diff
@@ -23,3 +23,16 @@ SS:
   - failed_alarm
   - lost_power_alarm
 
+EPM:
+  description: "Elevator position monitoring"
+  is_abstract: true
+  uses:
+  - floor_status
+
+BEA:
+  description: "Basic elevator alarms"
+  is_abstract: true
+  uses:
+  - fire_alarm
+  - failed_alarm
+  - max_load_alarm
```

Comment:
Is this important enough to be pulled in? If so it would be better as an optional point on the `ELV` general type.